READMEs/caching.md

```diff
@@ -0,0 +1,6 @@
+mac:
+`brew install redis` then `redis-server`
+
+this starts redis on your machine at 127.0.0.1:6379
+
+in a separate window you can now `yarn dev`
```

Comment:
Can we incorporate redis-server into our yarn commands, or does it need to be running prior or persistently across yarn commands?

Reply:
Will this work in Linux/WSL?

Reply:
this readme definitely needs some additional words, i forgot about it. 

I believe redis-server needs to be running prior, it can be a background service, though. I don't believe we want it persistently, only per-build (?)


It should work in Linux/WSL as long as homebrew is available (I believe it is?). Though maybe this is a good point towards running it in a docker container locally